README.md

```diff
@@ -65,7 +70,7 @@ $image = $poser->generate('license', 'MIT', '428F7E', 'plastic');
 echo $image->getStyle();
 ```
 
-The allowed styles are: `plastic`, `flat` and `flat-square`.
+The allowed styles are: `plastic`, `flat`, `flat-square` and `for-the-badge`.
```

Comment:
We miss a comma before "and" (I know, we missed it even before, but better late than never)